src/extract.ts

```diff
@@ -464,6 +497,55 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
     return ret;
 }
 
+function extractJuliaBenchmarkHelper(g: JuliaBenchmarkGroup, labels: string[] = []): BenchmarkResult[] {
+    const res: BenchmarkResult[] = [];
+    for (const key in g[1].data) {
+        const value = g[1].data[key];
+        if (value[0] === 'BenchmarkGroup') {
+            for (const x of extractJuliaBenchmarkHelper(value as JuliaBenchmarkGroup, [...labels, key])) {
+                res.push(x);
+            }
+        } else if (value[0] === 'TrialEstimate') {
+            const v = value as JuliaBenchmarkTrialEstimate;
```

Comment:
same as above. With the additional union type TS should infer that this is `TrialEstimate`

Reply:
I tried to remove the conversion, but it seemed to trigger a runtime error. So I kept it as it was here.